sdk/compiler/damlc/pkg-db/util.bzl

```
@@ -129,6 +129,10 @@ def _daml_package_rule_impl(ctx):
 
       cp -a {pkg_root}/* {iface_dir}
       cp -a $IFACE_DIR/{pkg_root}/* {iface_dir}
+
+      # we resolve symlink into actual file to avoid issues on MacOS
+      find {iface_dir} -type l -exec bash -c 'for link; do cp --remove-destination "$(readlink $link)" "$link"; done' _ {{}} +
```

I looked up whether this is the recommended method - apparently it is for GNU cp, but after reading through https://unix.stackexchange.com/questions/667204/cp-command-follows-symbolic-link-at-destination-and-clobbers-link-target-instead, it looks like it would be much simpler to just use rm instead, esp. considering you're already resorting to a bash invocation.
```suggestion
      find {iface_dir} -type l -exec bash -c 'for link; do rm "$link"; cp "$(readlink $link)" "$link"; done' _ {{}} +
```
It's up to you whether you care, this is a pretty minor problem.